_opensearch/common-parameters.md

```diff
@@ -14,4 +14,4 @@ Human-readable output | To convert output units to human-readable values (for ex
 Pretty result | To get back JSON responses in a readable format, add `?pretty=true` to the request URL. | `GET <index_name>/_search?pretty=true`
 Content type | To specify the type of content in the request body, use the `Content-Type` key name in the request header. Most operations support JSON, YAML, and CBOR formats. | `POST _scripts/<template_name> -H 'Content-Type: application/json`
 Request body in query string | If the client library does not accept a request body for non-POST requests, use the `source` query string parameter to pass the request body. Also, specify the `source_content_type` parameter with a supported media type such as `application/json`. | `GET _search?source_content_type=application/json&source={"query":{"match_all":{}}}`
-Stack traces | To include the error stack trace in the response when an exception is raised, add `error_trace=true` to the request URL. | `GET <index_name>/_search?error_trace=true`
+Stack traces | To include the error stack trace in the response when an exception is raised, add `error_trace=true` to the request URL. <br> <br> When the config setting `ttp.detailed_errors.enabled: false`, the request of `error_trace=true` will not contain the detailed error message.| `GET <index_name>/_search?error_trace=true`
```

Comment:
```suggestion
Stack traces | To include the error stack trace in the response when an exception is raised, add `error_trace=true` to the request URL. <br> <br> When the configuration setting `ttp.detailed_errors.enabled:` is set to `false` in opensearch.yml, `error_trace=true` will not contain a detailed error message.| `GET <index_name>/_search?error_trace=true`
```

Reply:
Is there a reason why "stack traces" is plural here? Normally, especially if the only option is error stack trace, it is singular.

Reply:
Also, please remove the colon in the suggestion: `ttp.detailed_errors.enabled:` should be `ttp.detailed_errors.enabled`

Reply:
@kolchfa-aws: To look at this from all angles, is it for when you're using this parameter in the context of a response that can return multiple indexes and thus multiple stack traces, such as `https://opensearch.org/docs/latest/opensearch/rest-api/document-apis/multi-get/`?

Still for changing it though.

Reply:
Ah ok. That makes sense. In that case, we can go either way. I would still change to singular though.